packages/multichain-account-service/package.json

```diff
@@ -51,7 +51,7 @@
     "@ethereumjs/util": "^9.1.0",
     "@metamask/accounts-controller": "^35.0.0",
     "@metamask/base-controller": "^9.0.0",
-    "@metamask/error-reporting-service": "^3.0.0",
+    "@metamask/error-reporting-service": "^3.0.1",
```

Comment:
Nit: Could you add a changelog entry for this bump

Reply:
There are no functional changes in this version, it just adds a deprecation notice. I can add an entry if you prefer, but I plan on removing `error-reporting-service` as a dependency soon too.